src/pytutor/src/test/runTest.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
```

Comment:
move test folder out `src`.

Reply:
Though we could do that, this is the folder structure generated by Yeoman Generator and also recommended [here](https://code.visualstudio.com/api/working-with-extensions/testing-extension)